mkw_stats_bot/mkw_stats/cogs/guild_cog.py

```
@@ -0,0 +1,662 @@
+                    inline=False
+                )
+                embed.add_field(
+                    name="🔧 Management",
+                    value=f"• Current channel: {channel.mention}\n• Change channel: `/setchannel #new-channel`\n• Check permissions: `/checkpermissions {channel.mention}`",
+                    inline=False
+                )
+                await interaction.response.send_message(embed=embed)
+            else:
+                await interaction.response.send_message("❌ Failed to set OCR channel. Please try again.", ephemeral=True)
+
+        except Exception as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)